### PR TITLE
Update days left logic and add spec for it

### PR DESCRIPTION
**Pull request by gunosk129**

### CODE REVIEW

File: app/models/subscription.rb [hunk 1]
```diff
@@ -64,7 +64,7 @@ def friends_joined?
 
   def days_left
     return MAX_DAYS if token_expire_at.nil?
-    [(token_expire_at - DateTime.now).to_i, 0].max
+    [(token_expire_at - DateTime.current).to_i / 1.day, 0].max
```
Comment (citizen428) [inline app/models/subscription.rb]: I think it's easier to do this:

```ruby
(token_expire_at.to_date - Date.current).to_i
```

Comment (gunosk129) [reply app/models/subscription.rb]: that's a lot easier

**Comment by hiattp** [comment]
ah I see what we're doing here. So the expiry is when the campaign ends, not when individual token expire. but I guess we can just do a `Subscription.update_all(token_expire_at: whenever)` when we know the date right?

**Comment by gunosk129** [comment]
@hiattp yes, that what we thought

**Pull request merged**